tests/repositories/test_installed_repository.py

```diff
@@ -56,29 +57,37 @@ def paths(self) -> dict[str, Path]:
         }
 
     @property
-    def sys_path(self) -> list[Path]:
-        return [ENV_DIR, SITE_PLATLIB, SITE_PURELIB]
+    def sys_path(self) -> list[str]:
+        return list(map(str, [ENV_DIR, SITE_PLATLIB, SITE_PURELIB]))
```

Comment:
Please rewrite this as a list comprehension 😉

Reply:
:rofl: